runtime/shared_common/UnitTest.hpp

```diff
@@ -44,7 +44,8 @@ class UnitTest {
 		PROTECT_NEW_ROMCLASS_DATA_TEST,
 		CACHE_FULL_TEST,
 		PROTECTA_SHARED_CACHE_DATA_TEST,
-		STARTUP_HINTS_TEST
+		STARTUP_HINTS_TEST,
+		OSCACHE_TEST,
```

Comment:
Trailing comma that wasn't there before.

Reply:
Fixed